iogt/settings/docker_compose_dev.py

```diff
@@ -11,7 +11,16 @@
         'INDEX_SETTINGS': {},
         'AUTO_UPDATE': True,
         'ATOMIC_REBUILD': True
-        }
+    }
+}
+
+
+WAGTAILTRANSFER_SECRET_KEY = os.environ.get('WAGTAILTRANSFER_SECRET_KEY')
+WAGTAILTRANSFER_SOURCES = {
+   'iogt_global': {
```

Comment:
Should we also make this configurable?

Reply:
I think we should because that is the easiest way to keep the secret key secret and changeable for each environment and site.

Reply:
I was referring to the `iogt_global` label.

Reply:
ah, right...

yes, I suppose we should, and I've updated the PR to reflect this.